hardware/Layout-Desing-Guide.md

```diff
@@ -0,0 +1,486 @@
+
+<details>
+
+<summary><font size=5>Table of Contents</font> </summary>
+ 
+ 
+ - [1. Why is the proper layout design important?](#1.-why-is-the-proper-layout-design-important?)
+ - [2. RF section](#2.-rF-section)
+    - [2.1. Different parts of RF section](#2.1.-Different-parts-of-RF-section)
+      - [2.1.1. Function of RF matching  and filtering network](#2.1.1.-Function-of-RF-matching-and-filtering-network)
+      - [2.1.2. Function of VDD filterings](#2.1.2.-Function-of-VDD-filterings)
+    - [2.2. Layout of RF section](#2.2.-Layout-of-RF-section)  
+      - [2.2.1. Layout of the matching network](#2.2.1.-Layout-of-the-matching-network)
+  - [3. Layout Design Practices](#3.-Layout-Design-Practices)
+    - [3.1. Layout Design Practices - 2.4 GHz matching network](#3.1.-Layout-Design-Practices---2.4-GHz-matching-network)
+    - [3.2. Layout Design Practices - Sub-GHz matching network](#3.2.-Layout-Design-Practices---Sub-GHz-matching-network)
+    - [3.3. Layout Design Practices - HFXO](#3.3.-Layout-Design-Practices---HFXO)
+    - [3.4. Layout Design Practices - VDD filtering](#3.4.-Layout-Design-Practices---VDD-filtering)
+    - [3.5  Layout Design Practices - General](#3.5.-Layout-Design-Practices---General)
+  - [4. Quiz](#4.-Quiz)
+
+</details>
+
+
+
+
+The purpose of this material is to help users design PCBs
+for the EFR32 Wireless Gecko Portfolio using design practices
+that allow for good RF performance.
+
+PCB layout requirements are described in the application note, [AN928.1: EFR32 Series 1 Layout Design Guide](https://www.silabs.com/documents/public/application-notes/an928.1-efr32-series1-layout-design-guide.pdf),  and the layout design process for EFR32 Series 1 Modules is discussed in [the datasheet of each module](https://www.silabs.com/support/resources.ct-data-sheets.ct-miscellaneous.p-wireless_bluetooth-low-energy_blue-gecko-bluetooth-low-energy-modules.p-wireless_bluetooth-low-energy_blue-gecko-bluetooth-low-energy-modules_bgm11s12f256ga-v2.p-wireless_bluetooth-low-energy_efr32bg1-series-1-modules.p-wireless_bluetooth-low-energy_efr32bg13-series-1-modules.sort=2,asc) and  besides that EFR32 reference design files are available in Simplicity Studio.
```

Comment:
Last sentence is too long with multiple "and" instances.  I suggest changing "each module and besides that" to "each module.  Besides that", removing the and to start a new sentence.